services/github/schemas/v1/taskcluster-github-config.v1.yml

```diff
@@ -36,6 +36,14 @@ properties:
       Automatically cancel running task group on a non-default branch.
       Useful for PRs that are updated frequently. Only the last build will be running.
     default: true
+  allowComments:
+    type: string
```

Comment:
Let's move this into policy section as a new property.

Reply:
yeah, I agree. I missed that we already have `policy.pullRequests` with very similar purposes. Moving it there